tools/ci_setup.ml

```diff
@@ -34,7 +34,6 @@ let forked_packages =
     ; "base_bigstring"
     ; "bin_prot"
     ; "bonsai_test"
-    ; "bonsai_web"
```

Comment:
Why don't you fix bonsai_web instead ?

Reply:
It no longer needs to be fixed, so I have removed it from the list of packages which I have modified. The Jane Street version will be downloaded instead.

Reply:
ok